src/data/contributors/nausherali.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Nausher Ali",
+    "skills": ["MERN", "Django", "Spring"],
+    "cause": "serve humanity ",
```

Comment:
What a great cause to code :) Just a small suggestion.
![image](https://user-images.githubusercontent.com/48255244/92311169-697a1300-efd2-11ea-94c9-9fabf2431e07.png)
I think `I code for serving humanity` is gramatically more accurate.